packages/adapter-neon/src/pg-types.ts

```diff
@@ -0,0 +1,226 @@
+import type neon from '@neondatabase/serverless'
```

Comment:
You already extracted this into its own file. Would it make sense to extract it into tis own file shared by both neon and pg?

Is it possible to do so via npm / pnpm at build time so both resulting packages contain a copy of the file, without the need to ship an extra package?